pkg/ccl/backupccl/restore_job.go

```diff
@@ -2010,51 +2018,98 @@ func insertStats(
 		return nil
 	}
 
-	totalNumBatches := len(latestStats) / restoreStatsInsertBatchSize
```

Comment:
I could see adding an early exit case for len(latestStats) == 0.  That would mean we don't update the job details, but we weren't doing that in the old case either. 🤷 Your call.

Reply:
good point, done